src/charm.lua

```diff
@@ -1090,7 +1120,9 @@ if next(SMODS.find_mod "MoreFluff") then
     local raw_CardArea_emplace = CardArea.emplace
     function CardArea:emplace(card, ...)
         local ret = { raw_CardArea_emplace(self, card, ...) }
-        if G.GAME and G.GAME.Bakery_charm == 'BakeryCharm_Bakery_Posterization' and card and card.config.center and card.config.center.set == 'Colour' then
+        if G.GAME and G.GAME.Bakery_charm == 'BakeryCharm_Bakery_Posterization'
+            and not is_charm_debuffed_safe() and card
+            and card.config.center and card.config.center.set == 'Colour' then
```

Comment:
This check doesn't work correctly when you use White during the Stoic, since the change is never retroactively applied.

Reply:
Resolved in bbaa626